packages/instantsearch.js/src/connectors/trending-facets/types.ts

```diff
@@ -0,0 +1,6 @@
+import type { TrendingFacetHit as Base } from 'algoliasearch/lite';
```

Comment:
you'll want to import this in the alogliasearch.d.ts file in the helper and correctly ts-ignore, and pass the relevant type of v3 (any?), v4 and v5